### ISSUE TITLE
Cannot load mjs test files

### ISSUE THREAD

**Issue by skanaar** [connor4312/nodejs-testing#1]
The extension does not seem to support `.mjs` esmodule files.

**Test output logs**
```
worker1> starting /Users/redacted/test-test/module.test.mjs
worker1> 1..0 # no tests found
```

Content of **module.test.mjs**
```
import { test } from 'node:test'
import { strictEqual } from 'node:assert'

test('case A', async () => {
  console.log('success')
})

test('case B', async () => {
  strictEqual(true, false)
})
```

I can successfully run this test file with `node --test`

**Comment by connor4312** [comment]
Thanks for the issue, I just published 1.0.1 with the fix. (May take a minute to appear on the marketplace)

**Comment by koistya** [comment]
@connor4312 I have `"type": "module"` in my `package.json`, using `.test.js` file extension. The list of tests shows up fine in the explorer, but when I run them, it fails with the error:

```
worker1> starting /Users/User/Projects/example/example.test.js
worker1> 1..0 # no tests found
```
